torch/_inductor/scheduler.py

```diff
@@ -3060,6 +3060,24 @@ def get_buffer_layout(self, buf_name: str) -> ir.Layout:
         assert buf.node is not None
         return buf.node.get_layout()
 
+    def update_zero_dim_cpu_tensor(self) -> None:
+        for node in self.nodes:
+            if node.node:
+                maybe_data = getattr(node.node, "data", None)
+                if (
+                    not isinstance(node.node, MultiOutput)
+                    and node.node.get_device().type == "cpu"
+                    and (
```

Comment:
It's surprising to me we have to skip `Pointwise`, or `MultiOutput`.

Maybe it would be easier as to check the reads of nodes which are cuda device -> 
V.graph.name_to_buffer[read_name], then check get_size and get_device ?